code/addons/test/src/node/vitest-manager.ts

```diff
@@ -55,6 +55,7 @@ export class VitestManager {
             clean: false,
             cleanOnRerun: !watchMode,
             reportOnFailure: true,
+            exclude: [...(this.vitest?.config?.coverage?.exclude || []), '**/*.stories.*'],
```

Comment:
`this.vitest` is _always_ going to be undefined at this point, either because this is the first time starting it, or becase a restart always closes vitest first.

So I don't think this is adequate to combine with the user's and/or Vitest's default `exclude` list.

We need to first resolve the Vitest config, and then we can read from it. The Vitest maintainers have given me a code snippet like this, that should work (maybe with slight modifications):

```ts
import { resolveConfig as viteResolveConfig } from 'vite';
import { VitestPlugin, resolveConfig as vitestResolveConfig } from 'vitest/node';

const resolveConfig = async () => {
  const viteConfig = await viteResolveConfig({
    plugins: [VitestPlugin()]
  });
  return vitestResolveConfig(viteConfig);
}
```

![image](https://github.com/user-attachments/assets/37069541-51a7-44e8-81ee-e161cbc96ba6)
![image](https://github.com/user-attachments/assets/388867a2-7579-4b8a-8835-7706ccbf17fb)